package-lock.json

```diff
@@ -3864,11 +3743,6 @@
       "resolved": "https://registry.npmjs.org/vscode-nls/-/vscode-nls-2.0.2.tgz",
       "integrity": "sha1-gIUiOAhEuK0VNJmvXDsDkhrqAto="
     },
-    "winreg": {
```

Comment:
do you expect remove winreg?

Reply:
check the upgrade, "vscode-extension-telemetry" is upgraded to 0.0.17, winreg is not required anymore.

Reply:
https://www.npmjs.com/package/npm-check-updates is a good tool for such version upgrade